domain_utils/domain_utils.py

```diff
@@ -236,3 +236,29 @@ def get_stripped_url(url, scheme=False, drop_non_http=False, use_netloc=True, ex
         loc_out=loc_out,
         path_out=path_out,
     )
+
+def get_scheme(url):
```

Comment:
As a note for myself: in the future we may want to pass in the same "default_to_" param as per `get_stripped_url` #14 discussions so that two columns of dataset can return consistent results when processing.